go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/ionos-cloud/terraform-provider-ionoscloud/v6
 go 1.20
 
 require (
+	github.com/gofrs/uuid v4.4.0+incompatible
```

Comment:
can we use v5 instead?

Reply:
@cristiGuranIonos done in 5369bab15a6a1015bc488c1140a42af0283ed025.